### DISCUSSION TITLE
How long it will take to access to academic benefits (72 hours passed)

### DISCUSSION THREAD

**Discussion by YoosufAathil** [community/community#63419] [GitHub Education]
Congrats! Your request for @YoosufAathil was approved and your benefits will be available soon.

There may be a wait period between verification and access to academic benefits.

**Comment by sasithwarnakafonseka** [comment]
@YoosufAathil normally how long they takes to verify your account as studnet

**Reply by YoosufAathil** [reply]
It will take within 2 to 3 hours to approve your identity as a student but you need wait 3 to 4 days to get access to the student benefits sometimes It will less than that!
